oml/metrics/embeddings.py

```diff
@@ -161,8 +161,17 @@ def setup(self, num_samples: int) -> None:  # type: ignore
 
         self.acc.refresh(num_samples=num_samples)
 
-    def update_data(self, data_dict: Dict[str, Any]) -> None:  # type: ignore
-        self.acc.update_data(data_dict=data_dict)
+    def update_data(self, data_dict: Dict[str, Any], indices: Optional[List[int]] = None) -> None:  # type: ignore
```

Comment:
Why are indices not obligatory? Do you want to do it in the future with better integration with datasets where order-keeping is important?

Reply:
yes, that's right
I've added `todo 522` comment to make sure I will not forget